addthis_social_widget.php

```diff
@@ -1251,7 +1251,12 @@ function addthis_display_social_widget($content, $filtered = true, $below_excerp
         global $addthis_styles, $addthis_new_styles, $post;
         global $addThisConfigs;
 
-        $styles = array_merge($addthis_styles, $addthis_new_styles);
+        if( is_array( $addthis_styles ) ) {
```

Comment:
I get the same warnings, but after fixing $addthis_styles I get warnings about $addthis_new_styles as well because it was also null. I wonder if it we can also fix by checking that $addthis_styles and $addthis_new_styles are arrays before the array _merge, if they're not then initialise them as empty arrays.